File: jvm/src/main/kotlin/com/jetbrains/signatureverifier/serialization/SequenceInfo.kt
package com.jetbrains.signatureverifier.serialization

import kotlinx.serialization.Serializable
import org.bouncycastle.asn1.ASN1Primitive

@Serializable
data class SequenceInfo(
  val content: List<EncodableInfo>
) : EncodableInfo {
  override fun toPrimitive(): ASN1Primitive =
    content.toPrimitiveDLSequence()
}

File: net/JetBrains.Serialization/src/SequenceInfo.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Org.BouncyCastle.Asn1;

namespace JetBrains.Serialization;

[JsonObject(MemberSerialization.OptIn)]
public class SequenceInfo : IEncodableInfo
{
  [JsonProperty("Content")] private List<IEncodableInfo> _content;

  [JsonConstructor]
  public SequenceInfo(List<IEncodableInfo> content)
  {
    _content = content;
  }

  public Asn1Encodable ToPrimitive()
  {
    return _content.ToPrimitiveDerSequence();
  }
}